contrib/synthetic_datagen/synthetic_datagen.py

```
@@ -0,0 +1,259 @@
+        nargs=2,
+        help="uniform distribution for output tokens",
+    )
+
+    args = parser.parse_args()
+
+    # Init random number generators
+    random.seed(args.seed) # We use python random for sampling corpus
+    rand = np.random.default_rng(seed=args.seed)
+
+    # Set the correct arguments based on specified distribution
+    if args.input_uniform != None:
```

Ruff (0.8.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)